referrals/repository.go

```diff
@@ -0,0 +1,9 @@
+package referrals
+
+import "context"
+
+type Repository interface {
+	HasReferralID(ctx context.Context, accountName string) (bool, error)
+	GetReferralID(ctx context.Context, accountName string) (string, error)
```

Comment:
can we remove `HasReferralID` and only have `GetReferralID` which return "" if the user doesn't have a referral?

Reply:
ok